### ISSUE TITLE
Add `deposit_end_time ` & `voting_start_time` to `Proposal` entity

### ISSUE THREAD

**Issue by DenisCarriere** [pinax-network/subgraph-cosmos-proposals#17]
- [x] update `Proposal` with any fields that seem useful
  - [x] `submit_time`
  - [x] `deposit_end_time`
  - [x] `voting_start_time`
  - [x] `voting_end_time`
  - [ ] ...
- [x] add `ProposalType` enum

https://buf.build/cosmos/cosmos-sdk/docs/main:cosmos.gov.v1#cosmos.gov.v1.Proposal
https://buf.build/cosmos/cosmos-sdk/docs/main:cosmos.gov.v1#cosmos.gov.v1.ProposalType

```proto
message Proposal {
	// id defines the unique id of the proposal.
	uint64 id = 1;
	// messages are the arbitrary messages to be executed if the proposal passes.
	repeated google.protobuf.Any messages = 2;
	// status defines the proposal status.
	ProposalStatus status = 3;
	// final_tally_result is the final tally result of the proposal. When
	// querying a proposal via gRPC, this field is not populated until the
	// proposal's voting period has ended.
	TallyResult final_tally_result = 4;
	// submit_time is the time of proposal submission.
	google.protobuf.Timestamp submit_time = 5 [
(gogoproto.stdtime) = true
];
	// deposit_end_time is the end time for deposition.
	google.protobuf.Timestamp deposit_end_time = 6 [
(gogoproto.stdtime) = true
];
	// total_deposit is the total deposit on the proposal.
	repeated cosmos.base.v1beta1.Coin total_deposit = 7 [
(gogoproto.nullable) = false,
(amino.dont_omitempty) = true
];
	// voting_start_time is the starting time to vote on a proposal.
	google.protobuf.Timestamp voting_start_time = 8 [
(gogoproto.stdtime) = true
];
	// voting_end_time is the end time of voting on a proposal.
	google.protobuf.Timestamp voting_end_time = 9 [
(gogoproto.stdtime) = true
];
	// metadata is any arbitrary metadata attached to the proposal.
	// the recommended format of the metadata is to be found here:
	// https://docs.cosmos.network/v0.47/modules/gov#proposal-3
	string metadata = 10;
	// title is the title of the proposal
	//
	// Since: cosmos-sdk 0.47
	string title = 11;
	// summary is a short summary of the proposal
	//
	// Since: cosmos-sdk 0.47
	string summary = 12;
	// proposer is the address of the proposal sumbitter
	//
	// Since: cosmos-sdk 0.47
	string proposer = 13 [
(cosmos_proto.scalar) = "cosmos.AddressString"
];
	// expedited defines if the proposal is expedited
	//
	// Since: cosmos-sdk 0.50
	// Deprecated: Use ProposalType instead.
	bool expedited = 14 [
deprecated = true
];
	// failed_reason defines the reason why the proposal failed
	//
	// Since: cosmos-sdk 0.50
	string failed_reason = 15;
	// proposal_type defines the type of the proposal
	//
	// Since: x/gov v1.0.0
	ProposalType proposal_type = 16;
}
```

```proto
enum ProposalType {
	// PROPOSAL_TYPE_UNSPECIFIED defines no proposal type, which fallback to PROPOSAL_TYPE_STANDARD.
	PROPOSAL_TYPE_UNSPECIFIED = 0;
	// PROPOSAL_TYPE_STANDARD defines the type for a standard proposal.
	PROPOSAL_TYPE_STANDARD = 1;
	// PROPOSAL_TYPE_MULTIPLE_CHOICE defines the type for a multiple choice proposal.
	PROPOSAL_TYPE_MULTIPLE_CHOICE = 2;
	// PROPOSAL_TYPE_OPTIMISTIC defines the type for an optimistic proposal.
	PROPOSAL_TYPE_OPTIMISTIC = 3;
	// PROPOSAL_TYPE_EXPEDITED defines the type for an expedited proposal.
	PROPOSAL_TYPE_EXPEDITED = 4;
}
```

**Comment by zolting** [comment]
`deposit_end_time` and `voting_end_time` are added to the schema, but nothing being pushed yet as the substreams store is not yet implemented

**Comment by zolting** [comment]
Fully implemented in #23
